app/routes/api.py

```diff
@@ -1,4 +1,7 @@
 from flask import Blueprint
+from app.models import mega_sena as model
```

Comment:
Evite imports com nome genéricos como `model`

```suggestion
from app.models.mega_sena import MegaSenaSchema
```